core/src/main/scala/org/apache/spark/deploy/rest/StandaloneRestServer.scala

```diff
@@ -135,7 +135,10 @@ private[rest] class StandaloneSubmitRequestServlet(
     val driverMemory = sparkProperties.get("spark.driver.memory")
     val driverCores = sparkProperties.get("spark.driver.cores")
     val driverExtraJavaOptions = sparkProperties.get("spark.driver.extraJavaOptions")
-    val driverExtraClassPath = sparkProperties.get("spark.driver.extraClassPath")
+    val commonExtraClassPath = sparkProperties.get("spark.common.extraClassPath")
+    val driverSpecificExtraClassPath = sparkProperties.get("spark.driver.extraClassPath")
+    val driverExtraClassPath = Option(( commonExtraClassPath ++ driverSpecificExtraClassPath)
```

Comment:
Ah ok, they can't be nulls since `get` returns an option (I thought it was a Java map)